spyder/plugins/variableexplorer/widgets/collectionseditor.py

```diff
@@ -26,7 +26,7 @@
 import warnings
 
 # Third party imports
-import cloudpickle
+from pympler.asizeof import asizeof

```

Comment:
I thought the name of the package was `asizeof`, but it's really pympler! That package is already present in conda-forge, so I just added it to our Anaconda.org channel so we can use it in our tests.

However, you also need to add it here to our setup.py, `requirements/conda.txt` and our Readme.